services/API-service/src/api/event/event.service.ts

```diff
@@ -169,15 +169,10 @@ export class EventService {
   ): SelectQueryBuilder<EventPlaceCodeEntity> {
     return this.eventPlaceCodeRepo
       .createQueryBuilder('event')
-      .select([
-        'area."countryCodeISO3"',
-        'event."eventName"',
-        'event."triggerValue"',
-      ])
+      .select(['area."countryCodeISO3"', 'event."eventName"'])
```

Comment:
@gulfaraz I am a bit hesitant if this is OK to just change, and doesn't have adverse affects in other scenarios. This was put in like this I think because of the multi-threshold work in e.g. UGA floods, where it is used to distinguish events of different magnitude? I guess as I'm writing this that they should already be separated by eventName (= glofasStation in that case). But did you check this in that scenario, and others?

Reply:
OK, I've tested the UGA floods case myself. And have also read up now on the other comments in this PR (e.g. regarding typhoon). I do think I can confirm this change is OK to merge, but I am afraid that it just solves a symptom of wider-spread disease. Because why is e.g. the typhoon being stored as something that can come out as trigger+warning? So we can either just merge it and treat this as step-by-step iterative solving, or we should dive deeper into it..